community-meetings/2021-05-11.md

```diff
@@ -22,10 +23,60 @@ Details on our ClusterAPI work so far and future plans
 ## Community Q&A
 Open Q&A / discussion
 
----
 
-# Minutes
-The minutes will be published here after the call concluded.
+# Call Minutes
+
+The meeting largely followed the [slide deck](2021-05-11-slides.pdf). After the presentation, community participants raised a total of 3 questions.
+
+1. The Flatcar team, Thilo, Sayan, Marga, Dongsu, Mathieu, Andy, Kai, and Iago introduce themselves.
+2. Andy briefly addresses the future of Flatcar following Kinvolk's acquisition by Microsoft.
+   - " [...] we want to assure the Flatcar community that Microsoft and the Kinvolk team will continue to collaborate with the larger Flatcar community on the evolution of Flatcar Container Linux.” - Brendan Burns, Microsoft
+   - " “This will not be a replay of the movie you’ve seen before. In fact, we and Microsoft are committed to doubling down on the Flatcar community: we want to expand the universe of partners, contributors, and users, to ensure a vibrant, successful and sustainable long-term future for Flatcar as a truly open, community-driven project.” - Chris Kühl, Kinvolk
+3. Thilo introduces monthly community calls and the new community focus of the Flatcar project, overcoming and leaving behind its single vendor past.
+4. Marga introduces the [interop matrix](../interop-matrix.md) as a means to track Flatcar's support of runtime environments (clouds, on premise, etc.).
+   - Some environments, while supported, do not currently have an owner.
+   - The project aims to have community owners who operate workloads / clusters in the respective environments, in the long term.
+5. Andy elaborates on Flatcar's core philosophy and shares details on stabilisation process and on release cadence.
+   - Alpha introduces new major versions, which then transition to Beta and Stable. Not every Alpha is promoted to Beta, not every Beta becomes Stable.
+   - Frequent Alpha releases, every 2 weeks on average.
+   - Beta release for (typically) every second Alpha, with patch releases in between.
+   - Stable releases roughly every 2 months.
+   - "golden" Stable to become LTS once a year.
+6. Sayan summarises the last round of releases, and provides an outlook of upcoming releases.
+   - April 28th round shipped new major Alpha (2857.0.0) and Beta (2823.1.0) versions, and patch level updates to Stable (2765.2.3) and to LTS (2605.15.1).
+     - Alpha release removes `rkt` and `kubelet-wrapper`. The changes will transition to Beta in June, and to Stable in July.
```

Comment:
Extra space before the `-` ?

Reply:
Yupp, it's a sub-item on the April 28th release.